### ISSUE TITLE
wireshark-dissector: Broken on Windows again

### ISSUE THREAD

**Issue by heinrich5991** [heinrich5991/libtw2#109]
Someone would need to test it there. Got some reports via Discord that it might not work anymore.

**Comment by LW-Davide** [comment]
Yes, it seems not working, I tried with Wireshark 4.2.9 (Old Stable Release) and it's not even loading the plugin

The error:
![8sgYUj3JRhB5](https://github.com/user-attachments/assets/2a6128dd-ccca-4093-87aa-9761858a3180)

(Built and tested on Windows)
